firmware/quadruna/VC/src/app/states/app_allStates.c

```diff
@@ -33,6 +34,9 @@ bool app_allStates_runOnTick100Hz(void)
     app_heartbeatMonitor_tick();
     app_heartbeatMonitor_broadcastFaults();
 
+    app_soc_update();
+    app_soc_broadcast();
```

Comment:
broadcast could likely be moved to 1Hz to save on bus load

Reply:
agreed, fixed